model/countries/FR/FR-parsing-rules.yaml

```diff
@@ -84,64 +86,73 @@ test_parsing_definitions:
     building: "1661"
 - id: "Test 5"
   type: street-address-alternative-1
-  input: "Appartement 36\n1661 Place Charles de Gaulle"
+  input: "1661 Place Charles de Gaulle\n Appartement 36"
   output:
-    street-address-alternative-1: "Appartement 36\n1661 Place Charles de Gaulle"
+    street-address-alternative-1: "1661 Place Charles de Gaulle\n Appartement 36"
     building-location: "1661 Place Charles de Gaulle"
     street: "Place Charles de Gaulle"
     building: "1661"
     address-overflow: "Appartement 36"
 - id: "Test 6"
   type: street-address-alternative-1
-  input: "Appartement 1\n58 rue du Gue Jacquet"
+  input: "58 rue du Gue Jacquet\nAppartement 1"
   output:
-    street-address-alternative-1: "Appartement 1\n58 rue du Gue Jacquet"
+    street-address-alternative-1: "58 rue du Gue Jacquet\nAppartement 1"
     building-location: "58 rue du Gue Jacquet"
     street: "rue du Gue Jacquet"
     building: "58"
     address-overflow: "Appartement 1"
 - id: "Test 7"
   type: street-address-alternative-1
-  input: "Appartement 36, 1661 Place Charles de Gaulle"
+  input: "1661 Place Charles de Gaulle, Appartement 36"
   output:
-    street-address-alternative-1: "Appartement 36, 1661 Place Charles de Gaulle"
+    street-address-alternative-1: "1661 Place Charles de Gaulle, Appartement 36"
     building-location: "1661 Place Charles de Gaulle"
     street: "Place Charles de Gaulle"
     building: "1661"
     address-overflow: "Appartement 36"
 - id: "Test 8"
   type: street-address-alternative-1
-  input: "Appartement 1, 58 rue du Gue Jacquet"
+  input: "58 rue du Gue Jacquet, Appartement 1"
   output:
-    street-address-alternative-1: "Appartement 1, 58 rue du Gue Jacquet"
+    street-address-alternative-1: "58 rue du Gue Jacquet, Appartement 1"
     building-location: "58 rue du Gue Jacquet"
     street: "rue du Gue Jacquet"
     building: "58"
     address-overflow: "Appartement 1"
 - id: "Test 9"
   type: street-address-alternative-1
-  input: "App 1, 58 rue du Gue Jacquet"
+  input: "58 rue du Gue Jacquet, App 1"
   output:
-    street-address-alternative-1: "App 1, 58 rue du Gue Jacquet"
+    street-address-alternative-1: "58 rue du Gue Jacquet, App 1"
     building-location: "58 rue du Gue Jacquet"
     street: "rue du Gue Jacquet"
     building: "58"
     address-overflow: "App 1"
 - id: "Test 10"
   type: street-address-alternative-1
-  input: "Appt 1, 58 rue du Gue Jacquet"
+  input: "58 rue du Gue Jacquet, Appt 1"
   output:
-    street-address-alternative-1: "Appt 1, 58 rue du Gue Jacquet"
+    street-address-alternative-1: "58 rue du Gue Jacquet, Appt 1"
```

Comment:
Can you drop the space after the comma? I would like to validate that case as well.

Reply:
Done.